src/Apps/Artist/Components/ArtistHeader.tsx

```diff
@@ -269,11 +328,115 @@ const handleOpenAuth = (mediator, artist) => {
   })
 }
 
+const RoundedFlex = styled(Flex)`
+  border-radius: 100px;
+`
+
+const renderAuctionHighlight = artist => {
+  if (
+    !artist.auctionResultsConnection ||
+    artist.auctionResultsConnection.edges.length < 1
+  ) {
+    return null
+  }
+  const topAuctionResult =
+    artist.auctionResultsConnection.edges[0].node.price_realized.display
```

Comment:
This would benefit from a `get` statement (using our internal helper). Might mean you can also get rid of/move the check above.